src/qt/clientmodel.cpp

```diff
@@ -153,6 +153,16 @@ QDateTime ClientModel::getLastBlockDate() const
     return QDateTime::fromTime_t(Params().GenesisBlock().GetBlockTime()); // Genesis block's time of current network
 }
 
+QString ClientModel::getLastBlockHash() const
+{
+    LOCK(cs_main);
+
+    if (chainActive.Tip())
+        return QString::fromStdString(chainActive.Tip()->GetBlockHash().ToString());
```

Comment:
```suggestion
    if (chainActive.Tip()) return QString::fromStdString(chainActive.Tip()->GetBlockHash().ToString());
```

Reply:
IMO it's ok-ish since it's kind of a copy-paste like thing of the function above.